.github/vale/dicts/aiven.dic

```diff
@@ -59,6 +60,7 @@ failover
 fileset
 filesets
 Flink
+Flink's
```

Comment:
Is this really neeeded?

Reply:
Changed the text in the content, I will remove this.